src/test/resources/no/skatteetaten/aurora/boober/service/AuroraDeploymentSpecRendererTest/reference-formatted-withDefaults.txt

```diff
@@ -1,57 +1,55 @@
+    path:              "actuator"                   // default
+    port:              "8081"                       // default
   }
   deployStrategy: {
-    type:              "rolling"                // default
-    timeout:           180                      // default
+    type:              "rolling"                    // default
+    timeout:           180                          // default
   }
-  artifactId:          "reference"              // fileName
-  groupId:             "no.skatteetaten.aurora" // reference.json
-  version:             "1"                      // reference.json
+  artifactId:          "reference"                  // fileName
+  groupId:             "no.skatteetaten.aurora"     // reference.json
+  version:             "1"                          // reference.json
   resources: {
     cpu: {
-      min:             "100m"                   // default
-      max:             "2000m"                  // default
+      min:             "100m"                       // default
+      max:             "2000m"                      // default
     }
     memory: {
-      min:             "128Mi"                  // default
-      max:             "512Mi"                  // default
+      min:             "128Mi"                      // default
+      max:             "512Mi"                      // default
     }
   }
-  replicas:            1                        // reference.json
+  replicas:            1                            // reference.json
   readiness: {
-    port:              8080                     // default
-    delay:             10                       // default
-    timeout:           1                        // default
+    port:              8080                         // default
+    delay:             10                           // default
+    timeout:           1                            // default
   }
-  liveness:            false                    // default
-  webseal:             false                    // default
-  debug:               false                    // default
-  pause:               false                    // default
-  alarm:               true                     // default
-  route:               true                     // reference.json
+  liveness:            false                        // default
+  webseal:             false                        // default
+  debug:               false                        // default
+  pause:               false                        // default
+  alarm:               true                         // default
+  route:               true                         // reference.json
   routeDefaults: {
-    name:              "reference"              // default
-    affiliation:       "aos"                    // default
-    env:               "utv"                    // default
-    separator:         "-"                      // default
+    host:              "@name@-@affiliation@-@env@" // default
```

Comment:
Should this have placeholders or not? Personally i feel that there should be placeholders here since we want to see what is configured not the replaced values

Reply:
See my comment below. 

Basically we first parse the header and add some of the values from there as placeholders